works.yaml

```diff
@@ -3646,7 +3614,7 @@ Your Face is a Saxophone:
   license:
   - CC0
   license_reference:
-  - http://plankhead.com/projects/yfias-source
+  - []
```

Comment:
see above note about use of [ ] syntax

In YAML, these two different representations mean the same thing:

    - one
    - two
    - three

    [one, two, three]

Thus, [ ] is just an empty list assigned to the key `license_reference`, while the dash-syntax is used for a list (often of only one item) of multiple elements.  If you try to use `- []` for an empty list, you'll end up with a list of one element, and that one element is an empty list, which is equivalent to doing this:

    [[]]

I hope that helps.